src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -264,10 +264,15 @@
              cofx)
             :dispatch [:wallet/update-gas-price true]))))
 
+(fx/defn navigate-after-transaction [{:keys [db] :as cofx} send-command? chat-id]
+  (if (= :wallet-send-transaction-modal (second (:navigation-stack db)))
+    (chat.models/navigate-to-chat cofx chat-id {})
+    (navigation/navigate-back cofx)))
+
 (handlers/register-handler-fx
  :close-transaction-sent-screen
- (fn [cofx [_ chat-id]]
+ (fn [cofx [_ send-command? chat-id]]
```

Comment:
what is the point of adding this parameter if it is not used in `navigate-after-transaction`?